index.html

```diff
@@ -46,13 +46,13 @@ <h2>Men Fen</h2>
       <div class="nav">
         <ul class="desktop-nav">
           <li>
-            <a href="#">Portfolio</a>
+            <a href="#work-section">Portfolio</a>
```

Comment:
- [ ] Your project's been approved...
**Optional suggestions 👍** 
-  You could check the functionality of your burger button,  and improve its performance of it, when clicking to be redirected to the contact form, there's a bug that shows the first part where we see "hi I'm a software developer" through all the page, instead of redirecting us to the contact form.  ♻
- There's no more to say, but it's time to merge ✔✨

Reply:
Thank you for your valuable suggestions. 🙇🏽‍♂️